django/http/request.py

```diff
@@ -261,7 +261,8 @@ def scheme(self):
                 )
             header_value = self.META.get(header)
             if header_value is not None:
-                return "https" if header_value == secure_value else "http"
+                header_values = [val.strip() for val in header_value.split(",")]
+                return "https" if secure_value == header_values[0] else "http"
```

Comment:
@felixxm I believe this takes the leftmost value from the split string.

Reply:
We can use the `maxsplit` argument:
```python
header_value, *_ = split(",", 1)
return "https" if header_value.strip() == secure_value else "http" 
```